packages/fast-components-styles-msft/src/utilities/high-contrast.ts

```diff
@@ -93,20 +108,45 @@ export const highContrastDoubleFocus: CSSRules<DesignSystem> = {
 };
 
 // Used to set 'selected-text' color
-export const highContrastSelection: CSSRules<DesignSystem> = {
+export const highContrastSelected: CSSRules<DesignSystem> = {
```

Comment:
this looks like a breaking change :bangbang:

Reply:
brought back the const and add deprecated comment

Reply:
if this is just a name change you can make the deprecated version simply point to the new version, like so:
`export const highContrastSelection: CSSRules<DesignSystem> = highContrastSelected;`

Reply:
updated